src/test/java/jalau/at18/katas/bankocr/adriana/BanckocrTest.java

```diff
@@ -0,0 +1,6 @@
+package jalau.at18.katas.bankocr.adriana;
+
+
+public class BanckocrTest {
```

Comment:
Pending?

Reply:
No really, It's because I refactor the class Banckocr and I take its methods to the classes Print and Printers, and in the same way the test go to these classes....

Reply:
ok, in those cases, you can remove classes like this if they are no more needed